linkerd/http/prom/src/record_response.rs

```diff
@@ -99,7 +72,7 @@ pub struct RequestCancelled(());
 pub struct NewRecordResponse<L, X, M, N> {
     inner: N,
     extract: X,
-    _marker: std::marker::PhantomData<fn() -> (L, M)>,
+    _marker: std::marker::PhantomData<(L, M)>,
```

Comment:
I believe `fn() -> T` allows covariance over T without triggering `dropck`. I _think_ we'd want to keep that, since this produces `(L, M)` via the extract instead of storing it.

Reply:
great observation, thank you! 04dfa43f5395f371bd817b40c8293d47a8087b10 reverts those changes :hammer: